docs/next-env.d.ts

```diff
@@ -2,4 +2,4 @@
 /// <reference types="next/image-types/global" />
 
 // NOTE: This file should not be edited
-// see https://nextjs.org/docs/basic-features/typescript for more information.
+// see https://nextjs.org/docs/app/building-your-application/configuring/typescript for more information.
```

Comment:
To rebase this branch on HEAD no? It was changed a while ago already. We shouldn't see this diff.